data/global.lua

```diff
@@ -112,6 +112,10 @@ table.contains = function(array, value)
 	return false
 end
 
+table.slice = function(source, from, to)
```

Comment:
It's better to call it an `array` since it only works with arrays. `source` sounds like it could work with anything.